addons/website_event/controllers/main.py

```diff
@@ -137,11 +137,16 @@ def dom_without(without):
 
         keep = QueryURL('/event', **{key: value for key, value in searches.items() if (key == 'search' or value != 'all')})
 
+        first_three_attendees_per_event = {}
+        for event in events:
+            first_three_attendees_per_event[event.id] = (',').join([registration.name.split(' ')[0] for registration in event.sudo().registration_ids[:3]])
+
```

Comment:
Remove that brol from here and template.
Maybe squash other changes into another "frontend" commit or rework commit message?